<launch>
  <node name="nvilidar_node"  pkg="nvilidar_ros"  type="nvilidar_node" output="screen" respawn="false" >
    <param name="serialport_name"             type="string" value="/dev/nvilidar"/>  
    <param name="serialport_baud"             type="int" value="512000"/>  
    <param name="ip_addr"                     type="string" value="192.168.1.200"/> 
    <param name="lidar_udp_port"              type="int" value="8100"/> 
    <param name="config_udp_port"             type="int" value="8200"/> 
    <param name="frame_id"                    type="string" value="laser_frame"/>
    <param name="resolution_fixed"            type="bool"   value="true"/>
    <param name="auto_reconnect"              type="bool"   value="true"/>
    <param name="reversion"                   type="bool"   value="false"/>
    <param name="inverted"                    type="bool"   value="false"/>
    <param name="angle_min"                   type="double" value="-180" />
    <param name="angle_max"                   type="double" value="180" />
    <param name="range_min"                   type="double" value="0.001" />
    <param name="range_max"                   type="double" value="64.0" />
    <param name="aim_speed"                   type="double" value="10.0"/>
    <param name="sampling_rate"               type="int" value="10"/>
    <param name="sensitive"                   type="bool" value="false"/>
    <param name="tailing_level"               type="int" value="6"/>
    <param name="angle_offset_change_flag"    type="bool" value="false"/>
    <param name="angle_offset"                type="double" value="0.0" />
    <param name="apd_change_flag"             type="bool" value="false"/>
    <param name="apd_value"                   type="int"  value="500"/>
    <param name="ignore_array_string"         type="string" value="" />
    <param name="filter_jump_enable"          type="bool"  value="true" />
    <param name="filter_jump_value_min"       type="int"  value="3" />
    <param name="filter_jump_value_max"       type="int"  value="50" />

  </node>
  <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser"
    args="0.19 0.0 0.2 0.06 0.0  0.0 /base_footprint /laser_frame 40" />
</launch>
   
